File: WebsiteTav/Models/Regular/TblStart.cs
namespace WebsiteTav.Models.Regular
{
    public class TblStart
    {
        public int id { get; set; }
        public string Name { get; set; }
        public string TellNo { get; set; }
        public string Description { get; set; }
        public string Brand { get; set; }
        public string SocialMediaLink { get; set; }
        public int Status { get; set; }

        public TblStart(int id)
        {
            this.id = id;
        }

		public TblStart(int id, string name, string tellNo, string description, string brand, string socialMediaLink, int status)
        {
            this.id = id;
            Name = name;
            TellNo = tellNo;
            Description = description;
            Brand = brand;
            SocialMediaLink = socialMediaLink;
            Status = status;
        }
        public TblStart(string name, string tellNo, string description, string brand, string socialMediaLink, int status)
        {
            Name = name;
            TellNo = tellNo;
            Description = description;
            Brand = brand;
            SocialMediaLink = socialMediaLink;
            Status = status;
        }

        public TblStart()
        {
            
        }
    }
}

File: WebsiteTav/Scripts/Admin/TblStart.js
function LoadInStarts()
{
    const table = document.getElementById('tblMain');

    //---> int    id
    //---> string Name
    //---> string TellNo
    //---> string Description
    //---> string Brand
    //---> string SocialMediaLink
    //---> int    Status
    const helps = SelectAllStarts();

    debugger;
    for (const entry of helps)
    {
        table.innerHTML +=
            `
                <tr>
                    <td>${entry.id}</td>
                    <td>${entry.Name}</td>
                    <td>${entry.TellNo}</td>
                    <td>${entry.Description}</td>
                    <td>${entry.Brand}</td>
                    <td>${entry.SocialMediaLink}</td>
                    <td>${entry.Status}</td>
                </tr>   
                
            `;
    }
}

LoadInStarts();